client/src/components/Tooltip.jsx

```diff
@@ -3,40 +3,47 @@ import React from 'react';
 import styles from './style.css.js';
 
 const Tooltip = (props) => {
+  const position = {
+    left: props.position[0],
+    top: props.position[1],
+  }
   const style = props.hover === props.id
-    ? Object.assign({}, styles.tooltipBody, styles.tooltipBodyActive)
+    ? Object.assign({}, styles.tooltipBody, styles.tooltipBodyActive, position)
     : styles.tooltipBody;
 
   return (
     <div style={style}>
       <p>Here's what people are saying:</p>
       <div className="summary" style={styles.reviewSummary}>
         <div>
-          <h4 style={styles.title}>{props.summary.good}%</h4>
+          <h4 style={styles.title}>{props.restaurant.review_summary.good}%</h4>
```

Comment:
```suggestion
          <h4 style={styles.title}>{props.restaurant.reviewSummary.good}%</h4>
```
Should be camelCased here and lines 25 & 31 to be consistent with your other variable names.

Reply:
This is the variable name in the database. I guess I will change it too then.

Reply:
I think it's fine for the database if your other variables look like it. But in your client it should be camelCased because everything else is camelCased.